doc/nrf/gs_recommended_versions.rst

```diff
@@ -22,6 +22,66 @@ However, there are some Zephyr features that are currently only available on Lin
 
    Before you start setting up the toolchain, install available updates for your operating system.
 
+.. _gs_supported_OS:
+
+Supported Operating Systems
+***************************
+
+The following table shows the operating systems and their various versions that supports the |NCS| tools:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Operating System
+     - x86
+     - x64
+     - ARM64
+   * - `Windows 11`_
+     - Tier 2
+     - Tier 2
+     - Not supported
+   * - `Windows 10`_
+     - Tier 2
+     - Tier 1
+     - Not supported
+   * - `Linux - Ubuntu 22.04 LTS`_
+     - Not supported
+     - Not supported
+     - Not supported
+   * - `Linux - Ubuntu 20.04 LTS`_
+     - Not supported
+     - Tier 1
+     - Not supported
+   * - `macOS 12`_
+     - Not applicable
+     - Tier 2
+     - Tier 2
+   * - `macOS 11`_
+     - Not applicable
+     - Tier 1
+     - Not supported
+   * - `macOS 10.15`_
+     - Not applicable
+     - Tier 3
+     - Not supported
+
+The table uses following three tiers to categorize the level of assistance each operating system provides:
+
+Tier 1
+  Tier 1 targets can be described as guaranteed to work.
```

Comment:
Are we talking about "build targets" here?
What are actually "Tier 1 build targets"?

Reply:
I understand that it is not about the building targets. Different tier levels refer to how thoroughly the tools are built, tested, and maintained.